sdk/storage/azure-storage-file/apireview/azure/storage/file/models.pyi

```diff
@@ -0,0 +1,180 @@
+# Stubs for azure.storage.file.models (Python 3)
```

Comment:
File should be hidden if interesting types are imported from azure.storage.file

Reply:
#7799